model/hash.go

```diff
@@ -5,21 +5,14 @@
 package model
 
 import (
-	"hash/fnv"
-	"io"
+	modelv1 "github.com/jaegertracing/jaeger-idl/model/v1"
 )
 
 // Hashable interface is for type that can participate in a hash computation
 // by writing their data into io.Writer, which is usually an instance of hash.Hash.
-type Hashable interface {
-	Hash(w io.Writer) error
-}
+type Hashable = modelv1.Hashable
 
 // HashCode calculates a FNV-1a hash code for a Hashable object.
 func HashCode(o Hashable) (uint64, error) {
```

Comment:
don't re-implement the functions, alias them
```suggestion
var HashCode = modelv1.HashCode
```